plextraktsync/trakt_list_util.py

```diff
@@ -51,7 +51,10 @@ def plex_items_sorted(self):
 
         https://github.com/Taxel/PlexTraktSync/pull/58
         """
-        _, items = zip(*sorted(dict(reversed(self.plex_items)).items()))
+        if len(self.plex_items) > 0:
```

Comment:
this optimization is not needed.

Reply:
I got this error with an empty Watchlist if this isn't there: 

```
INFO     Updating Plex List: Trakt Watchlist (0 items)                          
Traceback (most recent call last):
  File "/root/.local/bin/plextraktsync@1184", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/plextraktsync/cli.py", line 28, in wrap
    cmd(*args, **kwargs)
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/plextraktsync/commands/sync.py", line 57, in sync
    runner.sync(walker=w, dry_run=config.dry_run)
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/plextraktsync/sync.py", line 157, in sync
    self.update_playlists(listutil, dry_run=dry_run)
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/plextraktsync/sync.py", line 170, in update_playlists
    updated = self.plex.update_playlist(tl.name, tl.plex_items_sorted)
                                                 ^^^^^^^^^^^^^^^^^^^^
  File "/root/.local/pipx/venvs/plextraktsync@1184/lib/python3.11/site-packages/plextraktsync/trakt_list_util.py", line 57, in plex_items_sorted
    _, items = self.plex_items
    ^^^^^^^^
ValueError: not enough values to unpack (expected 2, got 0)
```

Reply:
hmm. but  then `return []`, and no else branch